ada_moveit/config/ada.srdf

```diff
@@ -256,4 +256,6 @@
     <disable_collisions link1="cameraMount" link2="uncalibrated_camera_link" reason="Adjacent"/>
     <disable_collisions link1="frontStabilizer" link2="enclosureBottom" reason="Adjacent"/>
     <disable_collisions link1="j2n6s200_link_6" link2="frontStabilizer" reason="Adjacent"/>
+    <disable_collisions link1="frontStabilizer" link2="enclosureTop" reason="Nearby, necessary because MoveIt Servo slows near collisions"/>
```

Comment:
If these will Never come into contact due to the geometry of the robot regardless of joint config, then the canonical reason is "Never", and you can comment more specific reasons if need be.

Same for the below